### PR TITLE
[docs] Add status header for azuremonitor exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for azuremonitor exporter,pls review :)

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]
/cc @pcwiese as code owner to verify beta stability

File: exporter/azuremonitorexporter/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Azure Monitor Exporter
 
-This exporter sends logs and trace data to [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/).
```
Comment (TylerHelmuth) [inline exporter/azuremonitorexporter/README.md:3]: This line should be moved under the new status header

Comment (Frapschen) [reply exporter/azuremonitorexporter/README.md:3]: done

**Review by pmm-sumo — APPROVED** [review]

**Comment by codeboten** [comment]
@pcwiese please have a look

**Pull request closed without merging**